### PR TITLE
Update AMIs for Kubernetes 1.19.2, 1.18.9, 1.17.12

### LINKED ISSUE #1965: Build new AMIs
1.19.2
1.18.9
1.17.12

### PR DESCRIPTION
**Pull request by randomvariable**
Signed-off-by: Naadir Jeewa <jeewan@vmware.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #1965 

/hold

Pending #1947

### CODE REVIEW

**Comment by randomvariable** [comment]
/unhold

ready for review

File: README.md [hunk 1]
```diff
@@ -94,12 +94,15 @@ Note: These AMIs are not updated for security fixes and it is recommended to alw
...
 |                          | v1.17.11                |
+|                          | v1.17.12                |
 | v1.18                    | v1.18.0                 |
 |                          | v1.18.1                 |
 |                          | v1.18.2                 |
 |                          | v1.18.8                 |
+|                          | v1.18.9                 |
 | v1.19                    | v1.19.0                 |
 |                          | v1.19.1                 |
+|                          | v1.19.12                |
```
Comment (CecileRobertMichon) [inline README.md:105]:
```suggestion
|                          | v1.19.2                |
```

**Review by vincepri — APPROVED** [review]
/approve

**Pull request merged**